sphinx/ext/apidoc.py

```diff
@@ -181,23 +181,16 @@ def is_skipped_package(dirname: str, opts: Any, excludes: List[str] = []) -> boo
         return True
 
     # Check there is some showable module inside package
-    if all(is_excluded(path.join(dirname, f), excludes) for f in files):
-        # all submodules are excluded
-        return True
-    else:
-        return False
+    return all(is_excluded(path.join(dirname, f), excludes) for f in files)
 
 
 def is_skipped_module(filename: str, opts: Any, excludes: List[str]) -> bool:
     """Check if we want to skip this module."""
     if not path.exists(filename):
         # skip if the file doesn't exist
         return True
-    elif path.basename(filename).startswith('_') and not opts.includeprivate:
-        # skip if the module has a "private" name
-        return True
-    else:
-        return False
+    # skip if the module has a "private" name
+    return path.basename(filename).startswith('_') and not opts.includeprivate
```

Comment:
I feel this refactoring divides this if-blocks to an early return condition (`path.exists()`) and the main condition.  Do you think that the main condition of "is_skipped_module()` is to check the filename is private or not? IMO, it's one of the conditions. So they should not be separated.

Reply:
hmm. If you wanted to return from within the if-else block in these sorts of cases, i'd be more inclined to go with a construction like the following:

```python
if (
    # skip if the file doesn't exist
    not path.exists(filename) or
    # skip if the module has a "private" name
    path.basename(filename).startswith('_') and not opts.includeprivate
):
    return True
else:
    return False
```